examples/cis3-nft-sponsored-txs/Cargo.toml

```diff
@@ -14,6 +14,7 @@ crypto-primitives = ["concordium-std/crypto-primitives"]
 [dependencies]
 concordium-std = {path = "../../concordium-std", default-features = false}
 concordium-cis2 = {path = "../../concordium-cis2", default-features = false}
+hex = "0.4.3"
```

Comment:
What is the size of this contract? I'm not sure we have other contracts with the hex dependency.

Reply:
cargo concordium build -e
smart contract module 172.766 kB

cargo concordium build
smart contract module 166.729 kB